README.md

```diff
@@ -43,9 +43,15 @@ The following must be installed and available in your `$PATH`:
 One simple way to run this without installing PhantomJS on your local machine is to use the included DockerFile.
 
 ```bash
-docker build -t ge-cancellation-checker .
+# If not already cloned:
+git clone https://github.com/davidofwatkins/ge-cancellation-checker.git
+cd ge-cancellation-checker
+
+# Run Docker
+git clone https://github.com/davidofwatkins/ge-cancellation-checker.git
```

Comment:
This should be removed, right? This is already done right above on line 47, right?